travis/getaddons.py

```diff
@@ -71,7 +71,7 @@ def get_addons(path):
         res = [path]
     else:
         res = [os.path.join(path, x)
-               for x in os.listdir(path)
+               for x in sorted(os.listdir(path))
```

Comment:
I would need some non trivial amount of help, as I'm totally unaware of how to setup a repo structure to work with `oca_dependencies.txt` at the moment... It's for me pretty much out of context right now, so I do not understand quite well where to start and what printscreen exactly you are expecting? Would it be ok for you, if I give you the following answer:

Before:
```
dependencies/enterprise/
dependencies/repo123/
dependencies/abc1231/
```
--addons-path [ODER UNKOWN]


After:
```
dependencies/enterprise/
dependencies/repo123/
dependencies/abc1231/
```
--addons-path dependencies/abc1231,dependencies/enterprise,dependencies/repo123
[ALPHANUMERICAL]

Do you really need a printscreen for this? I think I don't understand, actually, sorry. I hardly belief that this is what you want to see:

```
>>> sorted(['b', 'c', 'a'])
['a', 'b', 'c']
```

But what else is your concern? That's the only change introduced. I do not get it. Can you please explain?